rossum/password.py

```diff
@@ -35,8 +33,6 @@ def reset_command(ctx: click.Context, email: str) -> None:
     if email:
         with RossumClient(context=ctx.obj) as rossum:
             result = rossum.reset_user_password(email)
-            return click.echo(result.get("detail"))
+            click.echo(result.get("detail"))
     else:
-        raise click.ClickException(
-            "Please specify an email address to receive instructions for resetting it."
-        )
+        click.echo("Please specify an email address to receive instructions for resetting it.")
```

Comment:
When you do `password reset` without an email, the command will never run as arguments are required. This if condition is not needed.

Reply:
in case a user sets an empty string, this condition will work:
```rossum> password reset ""
Please specify an email address to receive instructions for resetting it.```

I can remove it, because anyway it will be checked over API:
```
Error: Invalid response [https://api.elis.develop.r8.lol/v1/auth/password/reset]: {"email":{"email":["Enter a valid email address."]}}
```

Reply:
upd: if a user will set empty string -> nothing will be sent to API:
```
rossum> password reset ""
rossum>
```

should I leave the else statement here?

Reply:
![image](https://user-images.githubusercontent.com/43633560/99244401-4f468780-2802-11eb-8c31-e8f0299a26db.png)

Reply:
I removed `if email` statement as well, thanks